backend/k8s-deployment.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gosmartkickers-deploy
+spec:
+  selector:
+    matchLabels:
+      app: gosmartkickers
+  template:
+    metadata:
+      labels:
+        app: gosmartkickers
+    spec:
+      containers:
+        - name: gosmartkickers
+          image: smartkickers/backend:latest
+          ports:
+            - containerPort: 3000
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 50m
+              memory: 20Mi
+            limits:
+              cpu: 500m
+              memory: 100Mi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: gosmartkickers
+spec:
+  type: LoadBalancer
```

Comment:
What was the reason to use `LoadBalancer` as type here? Does this workload need to be managed by some external LB? If the connection is closed in the cluster, then I would just set it to `ClusterIP`, so the only access to that workload is through ingress.

Reply:
It becomes unchanged when ingress was added to the configuration. I will change it to ClusterIP.